src/analysis/retail/evoker/preservation/modules/core/MasteryEffectiveness.tsx

```diff
@@ -266,6 +235,73 @@ class MasteryEffectiveness extends Analyzer {
     this.totalMasteryHealingDone += absorbFromMasteryBonusUsed;
   }
 
+  __masteryBoostCheck(event: ApplyBuffEvent | RefreshBuffEvent, existingAbsorb: number) {
+    const mastery = this.statTracker.currentMasteryPercentage;
+    const baseShieldValue = this.__calculateShield(event, existingAbsorb);
+    const lowerShieldBound = baseShieldValue * (1 + (mastery - 0.1));
+    const upperShieldBound = baseShieldValue * (1 + (mastery + 0.1));
+    if (event.absorb) {
+      if (lowerShieldBound <= event.absorb && event.absorb <= upperShieldBound) {
+        return true;
+      }
+    }
+    return false;
+  }
+  //TODO -- THESE CALCULATIONS ARE USING A +/- 10% THRESHOLD UNTIL ALL THE NEW RAID BUFFS ARE ADDED AND INCORPORATED INTO STATTRACKER
+  __calculateShield(event: ApplyBuffEvent | RefreshBuffEvent, existingAbsorb: number) {
+    let baseShielding = 0;
+    const vers = this.statTracker.currentVersatilityPercentage + 0.03; // TODO: REMOVE WHEN MARK OF THE WILD IS ADDED TO STATTRACKER
```

Comment:
imo you should just let this be 3% off until MotW is in. no risk of forgetting / missing that its gone in